test/stub_loader.js

```diff
@@ -4,6 +4,8 @@ const fs = require('fs');
 const assert = require('assert');
 const pirates = require('pirates');
 
+process.env["ESM_OPTIONS"] = '{ "cache": "node_modules/.cache/esm-stubbed"}';
```

Comment:
What do you think about just doing `process.env["ESM_DISABLE_CACHE"] = true` instead? In my quick test, it's a few seconds _faster_ for running a render test than when using a cache 🤷‍♂

Reply:
Here are the results I got when running
`> yarn test-render basic`

Disable Cache: 7.43s
Cache enabled (first run): 9.8s
Cache enabled (subsequent runs): 4.15s
Cache enabled (single file change): 5.33s

The numbers are averaged across 10 runs. During development, when debugging a render test, or when verifying changes - the cache makes a considerable difference.  

Making a change to just one or two files allows reusing the existing cache and is still a marked improvement over the disabled cache.

>In my quick test, it's a few seconds faster for running a render test than when using a cache 

Maybe the improvement does not show up when running the full integration test. 

As a developer I would always prefer the improved speed of the tests when I have to debug/develop new tests.